README.md

```diff
@@ -65,6 +65,9 @@ The configuration contains the following options:
 * `check_interval`: the frequency with which service checks will be initiated; defaults to `500ms`
 * `check_mocked`: whether or not health check is mocked, the host check always returns healthy and report up when the value is true
 * `checks`: a list of checks that nerve will perform; if all of the pass, the service will be registered; otherwise, it will be un-registered
+* `rate_limit_enabled` (optional): whether or not to enable rate limiting for reporting (defaults to `false`). If enabled, rate limiting is done via the [Token-Bucket algorithm](https://en.wikipedia.org/wiki/Token_bucket).
```

Comment:
can we create nested structure for the config? e.g. `{rate_limiting: {average_rate: 1, max_burst: 10, shadow_mode: true}}`